### ISSUE TITLE
Demangler coming up empty for some functions

### ISSUE THREAD

**Issue by bambu** [Vector35/binaryninja-api#535]
Not all symbols are properly demangled.  The following symbol results in a an empty name in the functions window as well as in the graph view. This is shown in the image below. The symbol for the function is `_ZTv0_n12_N13ostringstreamD1Ev`. In IDA it is called `virtual thunk to'ostringstream::~ostringstream()`

<img width="582" alt="screen shot 2016-11-01 at 6 52 42 am" src="https://cloud.githubusercontent.com/assets/564972/19887719/815d6484-a000-11e6-9be7-62eba40042b8.png">

**Comment by plafosse** [comment]
Funny thing is we demangle this symbol just fine its just applied incorrectly:
```>>> demangle_gnu3(Architecture['x86'], "_ZTv0_n12_N13ostringstreamD1Ev")
(<type: >, ['virtual_thunk_to_ostringstream::~ostringstream()'])```

**Comment by bambu** [comment]
This seems to have been resolved sometime before 1.786

**Comment by psifertex** [comment]
Thanks for catching that.
